### DISCUSSION TITLE
Intune Deployment

### DISCUSSION THREAD

**Discussion by syn3rgy01** [Raphire/Win11Debloat#129] [Q&A]
Hi,
 Love the script, I have defined my custom apps to remove and the switches to use however, Is there a way to deploy this via microsoft intune with the custom apps removal and switches?

thanks

**Comment by Raphire** [comment]
Heya,

I believe the best way is to package the script as an exe and run that. You will need a tool that includes the whole script directory, not just the script itself. I haven't used to tested any of these tools, but ps1 to exe seems to come up as a tool that would for this. Just beware that this method can cause issues with your AV.

Please let me know if you managed to get deployments through InTune working.

**Comment by LaunchPad46** [comment]
I'm interested in seeing about using this with Intune as well.
